sdlib/d/gc/page.d

```diff
@@ -728,6 +731,44 @@ private:
 					occupancyMask |= newOccupancy;
 					count += popCount(oldOccupancy ^ newOccupancy);
 
+					if (!ec.supportsMetadata) {
+						e.slabData.rawContent[i] = newOccupancy;
+						continue;
+					}
+
+					auto toRemove = (oldOccupancy ^ newOccupancy)
+						& e.slabMetadataFlags.rawNimbleAtomic(cast(uint) i);
```

Comment:
Make i a uint?

Reply:
With the new API of returning the reference to the raw static array, this is moot.